tests/torch/ptq/test_graphs.py

```diff
@@ -100,6 +101,8 @@ def test_min_max_classification_quantized_graphs(desc: ModelDesc, quantization_p
     quantization_parameters["advanced_parameters"] = AdvancedQuantizationParameters(disable_bias_correction=True)
     quantization_algorithm = PostTrainingQuantization(**quantization_parameters)
 
-    quantized_model = quantization_algorithm.apply(nncf_network, nncf_network.nncf.get_graph(), dataset=None)
+    quantized_model = quantization_algorithm.apply(
+        nncf_network, nncf_network.nncf.get_graph(), dataset=get_static_dataset(desc.input_sample_sizes, None, None)
+    )
```

Comment:
Looks like we're trying to introduce independent algorithm feature support here.
cc @alexsu52